src/modules/dummyqml/DummyQmlViewStep.cpp

```diff
@@ -28,8 +31,23 @@ DummyQmlViewStep::~DummyQmlViewStep() {}
 QString
 DummyQmlViewStep::prettyName() const
 {
-    return tr( "Example QML page." );
+    return m_notesName ? m_notesName->get() : tr( "Notes" );
 }
 
+void
+DummyQmlViewStep::setConfigurationMap( const QVariantMap& configurationMap )
+{ 
+    Calamares::QmlViewStep::setConfigurationMap( configurationMap ); // call parent implementation
+    
+    bool qmlLabel_ok = false;
+    auto qmlLabel = CalamaresUtils::getSubMap( configurationMap, "qmlLabel", qmlLabel_ok );
+    if ( qmlLabel_ok )
```

Comment:
If it's not ok, the returned map is empty, so it won't contain "notes". you can remove this check (or fold it with the nested if).